homeworks/Andrii.Morozov_AndriiMorozov88/8-css-variables/style_variables.css

```
@@ -0,0 +1,565 @@
+  font-family: var(--font-family-paragraf);
+  font-size: var(--font-size-large);
+  line-height: 1;
+  margin-bottom: 50px;
+}
+
+.design__title {
+  display: flex;
+  justify-content: center;
+}
+
+.design__text {
+  font-size: var(--font-size-title);
+  font-family: var(--font-family-title);
+  line-height: 1;
+  letter-spacing: -2px;
+}
+
+.design {
+  width: 100%;
+  margin-bottom: 100px;
+}
+
+.design__photo-container {
+  display: flex;
+  justify-content: space-around;
+}
+
+.archmove__title {
+  width: 40%;
+  display: flex;
+  justify-content: flex-end;
+  font-size: var(--font-size-title);
+  font-family: var(--font-family-title);
+  line-height: 1.25;
+  letter-spacing: -2px;
+}
+
+.archmove {
+  display: block;
+  margin-bottom: 100px;
+}
+
+.archmove__container {
+  display: flex;
+  justify-content: space-around;
+  padding-left: 80px;
+}
+
+.archmove__content {
+  display: block;
+  padding-top: 40px;
+}
+
+.archmove__icon {
+  padding-top: 15px;
+  fill: var(--color-n40);
+  stroke: var(--color-n40);
+}
+
+.archmove__icon--bird {
+  padding-left: 5px;
+}
+
+.archmove__block {
+  display: flex;
+  margin-bottom: 30px;
+  margin-left: 10px;
+}
+
+.archmove__block--bird {
+  display: flex;
+  margin-bottom: 30px;
+}
+
+.archmove__text {
+  font-family: var(--font-family-paragraf);
+  font-size: var(--font-size-large);
+  line-height: 1;
+  padding-left: 40px;
+}
+
+.archmove__button {
+  margin-left: 90px;
+  transition: transform .2s ease-in-out;
+}
+
+.archmove__button:hover {
+  transform: scale(1.2);
+  transition-duration: transform .2s ease-in-out;
+}
+
+.dream-project {
+  width: 100%;
+  background: var(--color-n13);
+}
+
+.dream-project__container {
+  display: flex;
+  justify-content: space-around;
+}
+
+.dream-project__content {
+  display: block;
+  padding-top: 200px;
+  padding-left: 100px;
+}
+
+.dream-project__button {
+  transition: .2s;
+}
+
+.dream-project__button:hover {
+  transform: scale(1.2);
+  transition: .2s;
+}
+
+.dream-project__text {
+  font-family: var(--font-family-title);
+  font-size: var(--font-size-title);
+  line-height: 1;
+  letter-spacing: -1px;
+  margin-bottom: 72px;
+  font-weight: var(--font-weight-bold);
+}
+
+.footer {
+  display: flex;
+  max-width: 100%;
+  height: 200px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.footer__menu {
+  display: flex;
+}
+
+.footer__list {
+  display: flex;
+  margin-left: 80px;
+}
+
+.footer__line {
+  list-style-type: none;
+  font-family: var(--font-family-paragraf);
+  font-size: var(--font-size-small);
+  margin-right: 48px;
+}
+
+.footer__logo {
+  margin-right: 180px;
+}
+
+@media (max-width: 770px) {
+
+  .header__content {
+    display: flex;
+    justify-content: center;
+  }
+
+  .menu__list {
+    display: block;
+  }
+
+  .header__buttons {
+    display: flex;
+  }
+
+  .header__logo {
+    display: none;
+  }
+
+  .logo {
+    display: none;
+  }
+
+  .dreamhouse__photo-container {
+    display: none;
+  }
```

```suggestion
  .header__logo,
  .logo,
  .dreamhouse__photo-container {
    display: none;
  }
```

Селектори можна писати через кому